README.md

```diff
@@ -28,51 +35,53 @@ to support GPU inference in the future through alternate backends.
 
 Currently, the following models are supported:
 
+- [BLOOM](https://huggingface.co/docs/transformers/model_doc/bloom)
```

Comment:
Can we list some interesting examples of these? BLOOM has BLOOMZ, GPT-J has GPT4All, etc

Reply:
I don't understand this request.

Reply:
So the README has

> Currently, the following models are supported:
> 
>     [BLOOM](https://huggingface.co/docs/transformers/model_doc/bloom)
>     [GPT-2](https://huggingface.co/docs/transformers/model_doc/gpt2)
>     [GPT-J](https://huggingface.co/docs/transformers/model_doc/gptj)
>     [GPT-NeoX](https://huggingface.co/docs/transformers/model_doc/gpt_neox): GPT-NeoX, StableLM, RedPajama, Dolly v2
>     [LLaMA](https://huggingface.co/docs/transformers/model_doc/llama): LLaMA, Alpaca, Vicuna, Koala, GPT4All, GPT4-X, Wizard
>     [MPT](https://www.mosaicml.com/blog/mpt-7b)

where GPT-NeoX and LLaMA have some examples of models that use those architectures. It would be nice to have some examples for BLOOM, GPT-2, GPT-J and MPT too (even if it's just "MPT" for the last one), just so that it doesn't look weirdly imbalanced.

They could also be links to the descriptions of the models, but I leave that up to you.